apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -120,6 +120,16 @@ type OpsRequestSpec struct {
 	// +optional
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.restoreFrom"
 	RestoreFrom *RestoreFromSpec `json:"restoreFrom,omitempty"`
+
+	// ttlSecondsBeforeAbort OpsRequest will wait at most TTLSecondsBeforeAbort seconds for start-conditions to be met.
+	// +kubebuilder:default=0
+	// +optional
+	TTLSecondsBeforeAbort int32 `json:"ttlSecondsBeforeAbort,omitempty"`
```

Comment:
should use `*int32` pointer for optional field

Reply:
fixed. and add comments to explain default behavior.